src/backend/joanie/settings.py

```diff
@@ -610,18 +615,22 @@ class Production(Base):
     CSRF_COOKIE_SECURE = True
     SESSION_COOKIE_SECURE = True
 
-    # For static files in production, we want to use a backend that includes a hash in
-    # the filename, that is calculated from the file content, so that browsers always
-    # get the updated version of each file.
-    STORAGES_STATICFILES_BACKEND = values.Value(
-        "whitenoise.storage.CompressedManifestStaticFilesStorage"
-    )
-
     # Privacy
     SECURE_REFERRER_POLICY = "same-origin"
 
     # Media
-    DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+    STORAGES = {
+        "default": {"BACKEND": "storages.backends.s3boto3.S3Boto3Storage"},
```

Comment:
```suggestion
        "default": {"BACKEND": "storages.backends.s3.S3Storage"},
```

`storages.backends.s3boto3.S3Boto3Storage` is an alias to `storages.backends.s3.S3Storage` and can be removed in a future major version. See : 

- https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#configuration-settings
- https://github.com/jschneier/django-storages/blob/d0dce62a8a35bd4c75f77d389781424d4f705c07/storages/backends/s3boto3.py